backend/contentBackend.js

```diff
@@ -238,20 +235,20 @@ async function getContentMetadata() {
  * site (including disabled content), along with
  * optionally defined "homeRoute".
  *
- * @param { string } - token used for identification
  *
  * @return {object} - metadata of content
  */
 /** @expose */
-export async function getSiteMetadata(jwt) {
-  await validateJWT(jwt);
+export async function getSiteMetadata() {
+  await checkIfValidDomain();
+
   return {
     contentMeta: await getContentMetadata(),
     homeRoute: await getHomeRoute(),
   };
 }
 
-/**
+/** 
```

Comment:
Nit: trailing whitespace. Not sure why this passes lint , it shouldn't. @rylandg do you know maybe?